src/main/java/com/softserve/messaging/config/KafkaConfiguration.java

```diff
@@ -1,6 +1,6 @@
-package com.softserve.message_sender.config;
+package com.softserve.messaging.config;
```

Comment:
side note: 
It's more appropriate to have package like: 
`group-id.artifact`
e.g.
`com.softserve.kickscooter.messaging..........`

Reply:
group-id -> `com.sofserve.kickscooter`
artifact -> `messaging`